src/Codegen/Generator.hack

```diff
@@ -250,8 +250,12 @@ final class Generator {
                 );
                 $objects = Vec\concat($objects, $this->getConnectionObjects($class, $class_fields[$class->getName()]));
             } elseif (!C\is_empty($class->getAttributes())) {
+                $fields = idx($class_fields, $class->getName());
```

Comment:
A better fix might be 

```hack
} elseif (C\contains_key($class_fields, $class->getName()) {
```

on Line 252.

Mind making that change and then I'll merge this?

Reply:
Would also be lovely to get a test for this, if you have the time. Just adding a class with a  `<<__ConsistentConstruct>>` annotation (and no other class or method-level annotations) to the test fixtures in `test/Fixtures/Playground.hack`, and ensuring nothing breaks, should be good.